tekton/resources/images/single-arch-template.yaml

```diff
@@ -76,6 +76,12 @@ spec:
               DESTINATIONS=""
               for tag in ${TAGS}; do DESTINATIONS="${DESTINATIONS} --destination ${IMAGE_URL}:${tag}"; done
               IFS=$OIFS
+
+              IMAGE_REGISTRY_USER=tekton-robot
+              IMAGE_REGISTRY_TOKEN=$(cat $IMAGE_REGISTRY_CREDENTIALS)
```

Comment:
NIT: $(cat "$IMAGE_REGISTRY_CREDENTIALS") is safer

Reply:
👍 updated